packages/core/tsconfig.json

```diff
@@ -30,7 +30,18 @@
     "paths": {
       "@clr/core/test/*": ["./src/test/*"],
       "@clr/core/*": ["./src/*"]
-    }
+    },
+    "plugins": [
+      {
+        "name": "ts-lit-plugin",
+        "strict": true,
+        "rules": {
+          "no-missing-import": "off",
+          "no-unknown-attribute": "off",
+          "globalAttributes": ["cds-layout", "cds-list", "cds-text"]
+        }
+      }
+    ]
```

Comment:
I disabled two of the rules for the lit-element linter. The `no-missing-import` is not picking up the `register` files. The `no-unknown-attribute` is not picking up any kebab case attributes. I think both of these are open issues so once the tool fixes them we can enable the checks.